tests/unit/Hooks/ImageHooksTest.php

```diff
@@ -92,7 +95,9 @@ public function testPostCreate() {
 		$imageHeight = 100;
 		$dimensions = array($imageWidth, $imageHeight);
 
-		$this->mockGetImageSize($this->imagehooks, $location, $dimensions);
+		$this->mockImageHooks = $this->mockPostCreate(['postCreate',]);
```

Comment:
Ohkay, I see. So, basically I can remove the mockGetImageSize method as it is not required.